ecosystem/python/sdk/aptos_sdk/transactions.py

```diff
@@ -207,14 +207,95 @@ def serialize(self, serializer: Serializer):
 
 
 class Script:
-    def __init__(self):
-        raise NotImplementedError
+    code: bytes
+    ty_args: List[TypeTag]
+    args: List[ScriptArgument]
+
+    def __init__(self, code: bytes, ty_args: List[TypeTag], args: List[bytes]):
```

Comment:
why not `List[ScriptArgument]` for `args` here?

Reply:
Sigh... I really wish Python typing did something automatically